spec/features/admin_claim_tasks_update_with_dqt_api_spec.rb

```diff
@@ -102,147 +101,1323 @@ def task_outcome
     perform_enqueued_jobs
   end
 
-  context "with submitted claim" do
-    let(:claim) do
-      claimant_submits_claim(
-        claim_attributes: {
-          date_of_birth: Date.new(1990, 8, 23),
-          first_name: "Fred",
-          national_insurance_number: "QQ100000C",
-          reference: "AB123456",
-          surname: "ELIGIBLE",
-          teacher_reference_number: "1234567"
-        }
-      )
+  context "with EarlyCareerPayments policy" do
```

Comment:
@nomasprime Please consider using describe here or at the one of the sub-levels. 6 layers deep of contexts is very hard to read/understand.
Nesting 6 levels deeps using only context causes exactly the same issue as nesting without using the alias of context. Yes it does infer meaning, but becomes lost this deeps. These are aliases for that reason.

The purpose of “describe” is to wrap a set of tests against one functionality while “context” is to wrap a set of tests against one functionality under the same state.